client/src/hooks/useOffsetForESL.ts

```diff
@@ -15,9 +20,11 @@ type handleApproveChangeShiftMutationType = UseMutationResult<
   IApproveESLOffsetInput,
   unknown
 >
+type handleApproveOffsetMutationType = UseMutationResult<any, unknown, IApproveOffsetInput, unknown>
 type HookReturnType = {
   handleAddNewOffsetMutation: () => NewOffsetFuncReturnType
   handleApproveEslOffsetMutation: () => handleApproveChangeShiftMutationType
```

Comment:
```suggestion
  handleApproveEslOffsetMutation: () => ApproveChangeShiftMutationType
```

Any types should start with capital letter and it should not be started with handle because this is not an action